src/coreclr/src/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1896,7 +1896,7 @@ private static void ThrowIfTypeNeverValidGenericArgument(RuntimeType type)
         internal static void SanityCheckGenericArguments(RuntimeType[] genericArguments, RuntimeType[] genericParamters)
         {
             if (genericArguments == null)
-                throw new ArgumentNullException();
+                throw new ArgumentNullException(nameof(genericArguments));
```

Comment:
`genericArguments` was never supplied by the user. As such, maybe this code ought to not throw at all (just let it throw NRE). But, not suggesting you change it.